### ISSUE TITLE
The react carbon component ToggleSmall doesn't have labels

### ISSUE THREAD

**Issue by xpander53** [carbon-design-system/carbon#2856]
## What package(s) are you using?
"carbon-components": "^10.2.0",
"carbon-components-react": "^6.114.1",

- [x] `carbon-components`
- [x] `carbon-components-react`

## Detailed description

> Describe in detail the issue you're having.

The react carbon component ToggleSmall doesn't have labels like the design.
https://www.carbondesignsystem.com/components/toggle/code

> Is this issue related to a specific component?
Yes ToggleSmall

> What did you expect to happen? What happened instead? What would you like to
> see changed?

The ToggleSmall should have the option to have on/off labels like the design. I can use Toggle and get the labels but it doesn't look exactly like ToggleSmall. Or I can use ToggleSmall but I would have to create and change my own labels.

> What browser are you working in?
Chrome

> What version of the Carbon Design System are you using?
v10

> What offering/product do you work on? Any pressing ship or release dates we
> should be aware of?

Watson Health for Clinical Trial Matching

## Steps to reproduce the issue

No steps - you can see the react implementation doesn't follow the design.
http://react.carbondesignsystem.com/?selectedKind=ToggleSmall&selectedStory=toggled&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybooks%2Fstorybook-addon-knobs

> Please create a reduced test case in CodeSandbox
> https://codesandbox.io/s/github/IBM/carbon-components/tree/master/examples/codesandbox

## Additional information

You can reach me on Slack -- jmaner@us.ibm.com / Joel Maner

- Screenshots or code
- Notes

**Comment by snidersd** [comment]
@dakahn Also see issue #2809 which is related to toggle labels. In addition, please add the A11y label and include it in the IBM Carbon Copy milestone. Thx!

**Comment by abbeyhrt** [comment]
This issue was solved through PR #2885 🎉
